src/lib/test/test_ecr_repos.py

```diff
@@ -28,17 +28,17 @@ def describe_instantiation():
     def it_instantiates(mock_ecr_interface):
       ecr_repos = ECRRepos(mock_ecr_interface)
       repo1 = ecr_repos._repos[0]
-      assert len(ecr_repos._repos) == 2
+      assert len(ecr_repos._repos) == 3
       assert isinstance(repo1, ECRRepo)
   def describe_all_repos():
     def it_returns_all_repos(mock_ecr_interface):
       ecr_repos = ECRRepos(mock_ecr_interface)
-      assert set(ecr_repos.all_repos()) == set(['foo2', 'foo1'])
-  def describe_get_repo():
+      assert set(ecr_repos.all_repos()) == set(['foo1', 'foo2', 'bar1'])
+  def describe_repos_with_name():
```

Comment:
the default behaviour of the test has changed quite a bit (exact matches vs contains) so would benefit from more thorough testing

Reply:
right on!